src/config/index.ts

```diff
@@ -96,6 +96,8 @@ export const OVERSEAS_ENTITY_REVIEW_PAGE = "overseas-entity-review";
 export const UPDATE_CHECK_YOUR_ANSWERS_PAGE = "update-check-your-answers";
 export const UPDATE_DUE_DILIGENCE_PAGE = "update-due-diligence";
 export const UPDATE_INTERRUPT_CARD_PAGE = "update-interrupt-card";
+export const UPDATE_MANAGING_OFFICER_INDIVIDUAL_PAGE = "update-managing-officer-individual";
```

Comment:
Please change this to `export const UPDATE_MANAGING_OFFICER_PAGE = "update-managing-officer";`

It's how I've defined it in UAR-149 and not having individual on it keeps it consistent with Register.

Reply:
Fixed